src/System.Private.CoreLib/shared/System/ComponentModel/DefaultValueAttribute.cs

```diff
@@ -80,173 +77,162 @@ bool TryConvertFromInvariantString(Type typeToConvert, string stringValue, out o
-        /// value.</para>
-        /// </devdoc>
+        /// <summary>
+        /// Initializes a new instance of the <see cref='System.ComponentModel.DefaultValueAttribute'/>
+        /// class using a <see cref='System.SByte'/> value.
+        /// </summary>
         [CLSCompliant(false)]
         public DefaultValueAttribute(sbyte value)
         {
             _value = value;
         }
 
-        /// <devdoc>
-        /// <para>Initializes a new instance of the <see cref='System.ComponentModel.DefaultValueAttribute'/> class using a <see cref='System.UInt16'/>
-        /// value.</para>
-        /// </devdoc>
+        /// <summary>
+        /// Initializes a new instance of the <see cref='System.ComponentModel.DefaultValueAttribute'/>
+        /// class using a <see cref='System.UInt16'/> value.
+        /// </summary>
         [CLSCompliant(false)]
         public DefaultValueAttribute(ushort value)
         {
             _value = value;
         }
 
-        /// <devdoc>
-        /// <para>Initializes a new instance of the <see cref='System.ComponentModel.DefaultValueAttribute'/> class using a <see cref='System.UInt32'/>
-        /// value.</para>
-        /// </devdoc>
+        /// <summary>
+        /// Initializes a new instance of the <see cref='System.ComponentModel.DefaultValueAttribute'/>
+        /// class using a <see cref='System.UInt32'/> value.
+        /// </summary>
         [CLSCompliant(false)]
         public DefaultValueAttribute(uint value)
         {
             _value = value;
         }
 
-        /// <devdoc>
-        /// <para>Initializes a new instance of the <see cref='System.ComponentModel.DefaultValueAttribute'/> class using a <see cref='System.UInt64'/>
-        /// value.</para>
-        /// </devdoc>
+        /// <summary>
+        /// Initializes a new instance of the <see cref='System.ComponentModel.DefaultValueAttribute'/>
+        /// class using a <see cref='System.UInt64'/> value.
+        /// </summary>
         [CLSCompliant(false)]
         public DefaultValueAttribute(ulong value)
         {
             _value = value;
         }
 
-        /// <devdoc>
-        ///    <para>
-        ///       Gets the default value of the property this
-        ///       attribute is
-        ///       bound to.
-        ///    </para>
-        /// </devdoc>
-        public virtual object Value
-        {
-            get
-            {
-                return _value;
-            }
-        }
+        /// <summary>
+        /// Gets the default value of the property this attribute is bound to.
+        /// </summary>
+        public virtual object Value => _value;
 
         public override bool Equals(object obj)
         {
             if (obj == this)
             {
                 return true;
             }
-
-            if (obj is DefaultValueAttribute other)
+            if (!obj is DefaultValueAttribute other)
```

Comment:
You need parens around the `obj is DefaultValueAttribute other`; otherwise this won't compile.

Reply:
Derp I swear I compiled - must not have saved